### ISSUE TITLE
Hieroglyphs on Cyrillic symbols

### ISSUE THREAD

**Issue by nautilusnsk** [messageformat/messageformat#337]
Compiled this lib to try some samples. "En" local works ok, but when I tried to use "ru" I got hieroglyphs instead of Cyrillic values.

```javascript
const mf = new MessageFormat('ru');
const msg = mf.compile("{arg, plural, =0 {нет яблок} one {одно яблоко} few {{arg} яблока} other {{arg} яблок}}");
var res = msg({arg: 0});
console.log(res);
```
Returns 
仆亠� �磡仍仂从
for me.

However this returns correct value (option for "zero" is without "я")
```javascript
const mf = new MessageFormat('ru');
const msg = mf.compile("{arg, plural, =0 {нет блок} one {одно яблоко} few {{arg} яблока} other {{arg} яблок}}");
var res = msg({arg: 0});
console.log(res);
```

Also next one returns correct value (option for "one" is empty)
```javascript
const mf = new MessageFormat('ru');
const msg = mf.compile("{arg, plural, =0 {нет яблок} one {} few {{arg} яблока} other {{arg} яблок}}");
var res = msg({arg: 0});
console.log(res);
```

So, can't get why these results are so inconsistent.

**Comment by eemeli** [comment]
I can't replicate this issue. Copy+pasting the first block of code outputs `нет яблок` for me. Maybe this is a version-dependent oddity? Or perhaps the original string you're working with has some malformed characters in it?

**Comment by nautilusnsk** [comment]
Ok, may be my flow of building was wrong.

What I did.
I took master (7c0ea87b1501925b8c1b6734ee0e3a59c25063d4)
npm install
npm run-script build

Took messageformat\packages\core\messageformat.js

Made a sample:
```html
<script src="messageformat.js"></script>
<script>
const mf = new MessageFormat('ru');
const msg = mf.compile("{arg, plural, =0 {нет яблок} one {одно яблоко} few {{arg} яблока} other {{arg} яблок}}");
var res = msg({arg: 0});
console.log(res);
</script>
```

And it logs me these hieroglyphs.

I attached my sample with compiled lib.

[messageformat-sample.zip](https://github.com/messageformat/messageformat/files/6860566/messageformat-sample.zip)
